pom.xml

```diff
@@ -27,6 +27,7 @@
 		<spring-cloud-commons.version>2.0.0.BUILD-SNAPSHOT</spring-cloud-commons.version>
 		<spring-cloud-config.version>2.0.0.BUILD-SNAPSHOT</spring-cloud-config.version>
 		<spring-cloud-stream.version>Elmhurst.BUILD-SNAPSHOT</spring-cloud-stream.version>
+		<spring-security-oauth2.version>2.2.1.RELEASE</spring-security-oauth2.version>
```

Comment:
This isn't managed by boot?

Reply:
Still a WIP : https://github.com/spring-projects/spring-security-oauth/issues/1240

Reply:
:+1:

Reply:
I don't know if there are plans to make this managed on boot. @rwinch any opinion on that? I don't need autoconfiguration, just the dependencies. We are talking about boot 2.0.

Reply:
@daniellavoie Thanks for reaching out! There are no plans for the old OAuth project version to be managed by Spring Boot.

Reply:
Had a chat with Rob, best option is to migrate `OAuth2RestTemplate` to `WebClient` using Spring Security 5. I'll reword that PR.

Reply:
Ok, it won't be as easy. Spring Security 5 doesn't support automatic token retrieval for resource servers. The old `OAuth2RestTemplate` handled that for us. @rwinch plans on integrating that for Security 5.1. We have 3 options:

* Wait for Security 5.1
* Integrate the old non-managed lib until migration to Security 5.1
* Write token retrieval logic with WebClient (could be base work for Security 5.1).

I think the most reasonable option is to wait for Security 5.1

@ryanjbaxter @spencergibb Any opinion?

Reply:
I dont see any reason why we need this PR right now so I dont see why we cant wait.